pkg/services/user/userimpl/user.go

```diff
@@ -209,14 +209,7 @@ func (s *Service) Delete(ctx context.Context, cmd *user.DeleteUserCommand) error
 }
 
 func (s *Service) GetByID(ctx context.Context, query *user.GetUserByIDQuery) (*user.User, error) {
-	user, err := s.store.GetByID(ctx, query.ID)
-	if err != nil {
-		return nil, err
-	}
-	if err := s.store.CaseInsensitiveLoginConflict(ctx, user.Login, user.Email); err != nil {
```

Comment:
so this is the change for `case_insensitive_login`?

Reply:
yes